src/foam/nanos/session/services/ClientLocalSettingsServiceImpl.js

```diff
@@ -0,0 +1,14 @@
+foam.CLASS({
```

Comment:
So none of this is required now, correct?

Reply:
yep. cleaned up. I don't know what I did wrong but DAO doesn't work the way it should ( DAO that used on put is not a Session context DAO ). I spent half of a day trying to figure out how to make it work but it was unsuccessful. I'd really appreciate your thoughts on my latest changes https://github.com/foam-framework/foam2/pull/3705/commits/8eba18631a66bcd017bb0821f3453afedcb8c964 . thanks!